lib/pg/pglogical/client.rb

```diff
@@ -219,15 +219,16 @@ def subscription_remove_replication_set(name, set_name)
       #     local_replication_lsn(Log Sequence Number)
       def subscription_show_status(name)
         sql = <<-SQL
-          SELECT sub.*, stat.remote_lsn AS remote_replication_lsn, stat.local_lsn AS local_replication_lsn
+          SELECT sub.*, stat.remote_lsn AS remote_replication_lsn, stat.local_lsn AS local_replication_lsn, stat.external_id
           FROM pglogical.show_subscription_status($1) sub
-          JOIN pg_replication_origin_status stat
+          LEFT JOIN pg_replication_origin_status stat
             ON sub.slot_name = stat.external_id
         SQL
 
         typed_exec(sql, name).first.tap do |s|
           s["replication_sets"] = s["replication_sets"][1..-2].split(",")
           s["forward_origins"] = s["forward_origins"][1..-2].split(",")
+          s["status"] = "corrupted" if s["external_id"].nil?
```

Comment:
I don't think this really helps. Plus I would rather not edit this if we don't have to. I think "down" is fine.

Reply:
yea, since we are on replication subscription than `down`  should represent subscription is down (not database), I will remove overriding status here